src/ops/conv.ts

```diff
@@ -365,11 +373,11 @@ export class ConvOps {
    *   - For more info, see this guide:
    *     [https://www.tensorflow.org/api_guides/python/nn#Convolution](
    *          https://www.tensorflow.org/api_guides/python/nn#Convolution)
-   * @param rates The dilation rates: `[rateHeight, rateWidth]` in which we
-   *     sample input values across the height and width dimensions in atrous
-   *     convolution. Defaults to `[1, 1]`. If `rate` is a single number, then
-   *     `rateHeight == rateWidth`. If it is greater than 1, then all values
-   *     of `strides` must be 1.
+   * @param dilations The dilation rates: `[dilationHeight, dilationWidth]`
+   *     in which we sample input values across the height and width dimensions
+   *     in atrous convolution. Defaults to `[1, 1]`. If `dilations` is a single
+   *     number, then `dilationHeight == dilationWidth`. If it is greater than
+   *     1, then all values of `strides` must be 1.
```

Comment:
The constraint "if dilations > 1 then strides must be equal to 1" is a limitation of the older indirect "space_to_batch -> conv -> batch_to_space" implementation approach in TF. This constraint should not apply and can be lifted in your current implementation, although typically the constraint is valid in most use cases.

Reply:
@gpapan 👍

Reply:
@gpapan the description of this param was there prior to this PR.   All that changed here was the param name.  I will fix it with the other PR that implements atrous convolution.

Reply:
Sounds good!